src/transformState.js

```diff
@@ -4,8 +4,28 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
 function transformState(state, actions) {
   // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties' :
+        Object.assign(state, action.extraData);
+        break;
+      case 'removeProperties' :
+        for (const keyToRemove of action.keysToRemove) {
+          if (state.hasOwnProperty(keyToRemove)) {
+            delete state[keyToRemove];
+          }
+        }
+        break;
+      case `clear` :
+        for (const s in state) {
```

Comment:
```suggestion
        for (const prop in state) {
```
Не называй переменные одной буквой, если это не счетчик в цикле)